src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -407,6 +407,9 @@ export class DropdownSearch extends React.Component<IDropdownSearchProps, {}> {
     }
 
     private isSearchOn(): boolean {
+        if (this.props.infiniteScroll) {
+            return true;
+        }
         return this.props.options.length > this.props.searchThresold;

```

Comment:
`return !!this.props.infiniteScroll || this.props.options.length > this.props.searchThresold;`